server/models/bill.coffee

```diff
@@ -2,17 +2,22 @@ cozydb = require 'cozydb'
 
 
 module.exports = Bill = cozydb.getModel 'Bill',
-    type: String
-    subtype: String
-    date: Date
-    vendor: String
-    amount: Number
-    plan: String
-    pdfurl: String
-    binaryId: String
-    fileId: String
-    content: String
-    isRefund: Boolean
+    type: String  # Main type of the bill (check already used types in konnectors)
+    subtype: String  # Subtype of the bill (check already defined subtypes)
+    date: Date  # Date of the billing
+    vendor: String  # Third party from which the bill originates
+    amount: Number  # Bill amount
+    vat: Number  # VAT part on the bill
+    currency: String  # Currency of the bill
+    plan: String  # TODO
```

Comment:
Only use case is in [https://github.com/cozy-labs/konnectors/blob/23c8e3ef4354ef94e876e296695702df3a6a5fbc/server/konnectors/github.coffee](Github connector). Not sure what it should be set to, and looks like it could be a duplicate with other fields.